lib/ui/widgets/post_content/index.dart

```diff
@@ -37,4 +42,19 @@ class PostContent extends StatelessWidget {
       ],
     );
   }
+
+  List<String> _isLinkInContent() {
```

Comment:
I think that it would be better to name this correctly based on the return type. Something like `_getUrisToPreview` would be self explanatory. What do you think?